recipe/build.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
-if [[ $(uname) == 'Darwin' ]]; then
+if [[ $(uname) == Darwin ]]; then
   export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
-elif [[ $(uname) == 'Linux' ]]; then
```

Comment:
We don't have too. It is indifferent. (But my text editor syntax highlight likes it that way :wink:)